public/app/features/alerting/unified/rule-list/PaginatedDataSourceLoader.tsx

```diff
@@ -72,12 +67,12 @@ export function PaginatedDataSourceLoader({ rulesSourceIdentifier, application }
             ))}
           </ListSection>
         ))}
-        <LazyPagination
-          nextPage={nextPage}
-          previousPage={previousPage}
-          canMoveForward={canMoveForward}
-          canMoveBackward={canMoveBackward}
-        />
+        {hasMoreGroups && (
+          // this div will make the button not stretch
+          <div>
+            <LazyPagination loadMore={fetchMoreGroups} />
```

Comment:
Should we rename the `LazyPagination` component? It no longer provides pagination, it's only a button

Reply:
I'll inline this too